docs/README.md

```diff
@@ -3,3 +3,61 @@
 Caltrans Performance Measurement System
 
 PeMS is open-source software that is designed, developed, and maintained by [Compiler LLC](https://compiler.la) on behalf of Caltrans Traffic Operations.
+
+## Current Work
+We do sprint planning and track day-to-day work in our [Project Board](https://github.com/orgs/compilerla/projects/3/views/1).
```

Comment:
Nitpick, the Markdown linter (software that checks formatting) is complaining because there is supposed to be a blank line between headings and the next paragraph.